app/features/hit_fox_feed.rb

```diff
@@ -0,0 +1,23 @@
+require 'entities/github_feed'
```

Comment:
it is actually in lib/entities :-)

i wasn't really sure about putting it there or in in the models directory.